styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -92,16 +92,16 @@
 }
 
 .hostingOptions-logo {
-  @include make-sm-column(5);
-  @include make-sm-column-push(3.5);
+  @include make-sm-column(3.5);
```

Comment:
I understand the need for push's to have fractional offsets, when dealing with an odd number of columns, but do we really want columns themselves to not adhere to sizing standards? That doesn't seem like a good idea to me, and could start being treated as precedent, which I don't think we want.